### PR TITLE
Improvements to the O3 setup using Docker recipe

### PR DESCRIPTION
**Pull request by denniskigen**
Improvements to the O3 setup guide using Docker learned from a practical session with @ibacher and @NethmiRodrigo.

### CODE REVIEW

File: pages/docs/recipes/set-up-o3-instance.en-US.mdx [hunk 1]
```diff
@@ -89,37 +94,54 @@ You might want to use Docker if:
...
 
 ### Step 2
 
-Launch [Docker Desktop](https://docs.docker.com/desktop/) or the Docker daemon.
+Launch [Docker Desktop](https://docs.docker.com/desktop/) or [Docker Compose (v2)](https://docs.docker.com/compose/install/).
 
 ### Step 3
 
-Run `docker compose build` from the root of the repository (You might need to run `docker-compose build` instead if you're using `docker-compose`).
+Next we want to install the latest version of O3. To do that, we'll need to figure out what tag to use. Run `git tag --sort=-v:refname` to get a list of tags sorted by version number. At the time of writing, the latest tag is `3.0.0-beta.20`. To use this tag, run the following command:
```
Comment (ibacher) [inline pages/docs/recipes/set-up-o3-instance.en-US.mdx:117]: Nice one!

**Review by ibacher — APPROVED** [review]
LGTM. Might it be worth including [this command](https://github.com/openmrs/openmrs-esm-core/blob/01c32c2090a47a881d5f9fa39650ec4a9037aa36/.github/workflows/e2e.yml#L82) that we use in e2e tests to wait for the backend to come up?

```sh
while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost/openmrs/login.htm)" != "200" ]]; do sleep 10; done
```

I think it would also work with `http://localhost/openmrs/health/started`.

**Review by NethmiRodrigo — APPROVED** [review]
Very helpful update. Thanks @denniskigen!

**Comment by denniskigen** [comment]
> http://localhost/openmrs/health/started

Sure. I don't know how best to work either of those in. Could you suggest a change?

**Comment by denniskigen** [comment]
Merging for now. I'll find a way to work in your suggestion, @ibacher.

**Pull request merged**